### PR TITLE
RestClient has been removed.

### PR DESCRIPTION
**Pull request by tgxworld**
@smasry @oblakeerickson Please review :)

### CODE REVIEW

File: plugin.rb [hunk 1]
```diff
@@ -73,19 +71,24 @@ def fetch_user_details(auth_result, token, id)
...
+
+    response = Excon.get(
+      user_json_url,
+      headers: {
+        "Authorization" => "Bearer #{token}",
+        'Accept' => SiteSetting.oauth2_token_accept
+      }
+    )
+
+    if response.code != 200
```
Comment (smasry) [inline plugin.rb]: excon uses `status` instead of code

File: plugin.rb [hunk 2]
```diff
@@ -73,19 +71,24 @@ def fetch_user_details(auth_result, token, id)
...
+      auth_result.failed = true
+
+      if response.code == 401
```
Comment (smasry) [inline plugin.rb]: See the above comment

File: plugin.rb (hunk 1 above)
Comment (tgxworld) [reply plugin.rb]: oops you're right :) Will fix.

**Comment by tgxworld** [comment]
@smasry Thank you fore reviewing.

**Review by smasry — APPROVED** [review]

**Pull request merged**